launcher/Application.cpp

```diff
@@ -1633,3 +1633,17 @@ QString Application::getUserAgentUncached()
 
     return BuildConfig.USER_AGENT_UNCACHED;
 }
+
+int Application::suitableMaxMem()
+{
+    float totalRAM = (float)Sys::getSystemRam() / (float)Sys::mebibyte;
+    int maxMemoryAlloc;
+
+    // If totalRAM < 6GB, use (totalRAM / 1.5), else 4GB
+    if (totalRAM < (4096 * 1.5))
+        maxMemoryAlloc = (int) (totalRAM / 1.5);
```

Comment:
Wouldn't it make more sense to divide by 2? (e.g. if I have 4gb of ram, I'd want to allocate just 2)

Reply:
These are just defaults. People can reduce the RAM they want to allocate. But yea, I'll wait for more people to chime in.

Reply:
There was another user in discord suggesting the divide by two and letting the default max be 12GB

I'm personally fine with both of those suggestions

Reply:
let's have the default max at 8GB. 12 seems a bit too overkill.

Reply:
How about simply:

```c++
int maxMemoryAlloc = std::min(int(totalRAM / 2), 4096);
```

Reply:
let's settle at 6gb. ~~also i don't know programming but could totalRAM be not integer since we measure it down to megabytes~~

Reply:
Thanks for y'all's inputs. We'll merge it as-is for now.